cfme/configure/access_control.py

```diff
@@ -1228,7 +1229,12 @@ class TenantQuotaView(ConfigurationView):
     def is_displayed(self):
         return (
             self.form.template_cb.is_displayed and
-            self.title.text == 'Manage quotas for Tenant "{}"'.format(self.context['object'].name)
+            any((
```

Comment:
**Required**

Use the contextual object to determine the correct title here (Tenant vs Project).  This is not a unique identification if we're on the correct page.